Makefile

```diff
@@ -161,6 +168,19 @@ container:
 container-push:
 	docker push ${IMG}
 
+.PHONY: container-target-allocator-push
+container-target-allocator-push:
+	docker push ${TARGET_ALLOCATOR_IMG}
+
+.PHONY: container-target-allocator
+container-target-allocator: GOOS = linux
+container-target-allocator: targetallocator
+	docker buildx build --load --platform linux/${ARCH} -t ${TARGET_ALLOCATOR_IMG}  cmd/cwa-allocator
+
+.PHONY: ta-build-and-push
+ta-build-and-push: container-target-allocator
+ta-build-and-push: 	container-target-allocator-push
```

Comment:
nit: spacing

Reply:
Thats how you run two commands back to back in makefile. [For example](https://github.com/aws/amazon-cloudwatch-agent-operator/blob/099460aea6622b73557017a14b5c46e1b10de680/Makefile#L74)

Reply:
Isn't that the same as
```
ta-build-and-push: container-target-allocator container-target-allocator-push
```

Reply:
I was just following the other examples in our makefile but either one works for me